### PR TITLE
[Translation zh-tw] Udacity > ud890

### PR DESCRIPTION
**Pull request by kennith**
@limhenry, please review.

[Check out a review request at GitLocalize](https://gitlocalize.com/repo/107/zh-TW/review/438)

### CODE REVIEW

**Comment by sotayamashita** [comment]
I need to remove extra code

**Comment by limhenry** [comment]
Looks good for me!

**Comment by kennith** [comment]
@sotayamashita I will pause the translation from GitLocalize until the extra code is removed. Where can I get the latest update?

File: src/content/zh-tw/_shared/udacity/ud890.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN" "http://www.w3.org/TR/REC-html40/loose.dtd">
```
Comment (petele) [inline src/content/zh-tw/_shared/udacity/ud890.html]: has extra doctype and html elements

**Comment by sotayamashita** [comment]
@kennith Sorry about that. We will fix the issue until 12/14. :bow:

**Comment by petele** [comment]
@sotayamashita - any updates on this?

**Comment by sotayamashita** [comment]
@petele Sorry not to update. I fixed the issue and it will be deploy today. :bow:

**Comment by sotayamashita** [comment]
@kennith I have create a PR to remove extra code. Please review it

**Pull request merged**